tools/data_usage/setup.md

```diff
@@ -0,0 +1,41 @@
+# Setting up eBPF for Data Usage
+
+## Step 1: eBPF Installation
+Follow the [instructions](https://github.com/microsoft/ebpf-for-windows/blob/master/docs/GettingStarted.md#Prerequisites) for "Prerequisites", "How to clone and build the project" and "Installing eBPF for Windows" sections. 
+For the "How to clone and build the project" section instead of:
+> 1. ```git clone --recurse-submodules https://github.com/microsoft/ebpf-for-windows.git```
+
+Run the following:
+> ```git clone --recurse-submodules https://github.com/trishms/ebpf-for-windows.git```
+
+## Step 2: VM Installation Instructions
+Follow the [instructions](https://github.com/microsoft/ebpf-for-windows/blob/master/docs/vm-setup.md) for the "One-Time Setup" and "Installing eBPF into a VM". You'll need this VM to load eBPF, the necessary eBPF programs and run the user mode application.
+
+## Step 3: Prep and demo
+
+1. Set up the VM (Refer to Step 2)
+2. Power it off and in that VM's settings, go to Processor and change the number of processors to 1. (This is because Per-CPU maps are not yet implemented).
```

Comment:
Per-cpu map is now implemented I guess? It's not there when we implemented our project.